### ISSUE TITLE
unable to build Netbeans from within Netbeans

### ISSUE THREAD

**Issue by wumpz** [apache/netbeans#7872]
### Apache NetBeans version

Apache NetBeans 23

### What happened

I cloned the Netbeans repository and started from within Netbeans **ant build**.

Then the build stops at 

https://github.com/apache/netbeans/blob/01a871f055428abdf37d5b5d09999909e5825a48/java/api.debugger.jpda/build.xml#L29

this line with the message, that **com/sun/jdi/AbsentInformationException.class** was not found.

I tested this using JDK 17, JDK 21 but with Netbeans internal Ant. So far I did not use a separate Ant installation.

How can I tackle this problem? How to find my missconfiguration?

### Language / Project Type / NetBeans Component

Netbeans itself

### How to reproduce

1. I cloned the Netbeans repository
2. right click on build.xml and **ant build**.

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Windows 11

### JDK

17, 21

### Apache NetBeans packaging

Apache NetBeans provided installer

### Anything else

This happens every time. 

### Are you willing to submit a pull request?

No

**Comment by mbien** [comment]
yeah the (full) build has some issues when started from within NB (the jpda module specifically too). The build instructions are in the readme.
Short version is that you build it once in the console, then you open the modules you are interested in and edit/launch/debug/rebuild from there.

**Comment by wumpz** [comment]
So maybe at least the readme.md should reflect that ...

**Comment by mbien** [comment]
it does. If you follow the readme NB will build. The tip how to work with modules in NB is there too in the notes section.

**Comment by wumpz** [comment]
So then its my missunderstanding. Thx for the help.

**Comment by mbien** [comment]
the issue itself is still valid. I think we should keep it open in case someone would like to fix this.
